### PR TITLE
chore: add slack channel for csi-secrets-store

### PR DESCRIPTION
**Pull request by aramase**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, using `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when the PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -254,6 +256,14 @@ Job config for test jobs run for each PR in prow can be found [here](https://git
   kubectl logs csi-secrets-store-secrets-store-csi-driver-7x44t secrets-store
   ```
 
+## Communication
```
Comment (ritazh) [inline README.md:259]: How about we add this to the top of the readme before `#Features`, something like this?

```
## Want to help?
Join us to help define the direction and implementation of this project!
- Join the [`#csi-secrets-store`]((https://kubernetes.slack.com/messages/csi-secrets-store)
  channel on [Kubernetes Slack](https://kubernetes.slack.com/).
- Use [GitHub Issues](https://github.com/kubernetes-sigs/secrets-store-csi-driver/issues)
  to file bugs, request features, or ask questions asynchronously.
```

Comment (aramase) [reply README.md]: updated!

File: README.md [hunk 2]
```diff
@@ -30,7 +36,8 @@ The Secrets Store CSI driver `secrets-store.csi.k8s.io` allows Kubernetes to mou
   - [Unit Tests](#unit-tests)
   - [End-to-end Tests](#end-to-end-tests)
 - [Known Issues and Workarounds](#known-issues-and-workarounds)
-- [Contributing](#contributing)
+- [Troubleshooting](#troubleshooting)
```
Comment (ritazh) [inline README.md:39]: While you are in here, pls remove `Known Issues and Workarounds` altogether

Comment (aramase) [reply README.md:38]: Updated!

**Pull request merged**